pkg/reconciler/dependencybuild/dependencybuild.go

```diff
@@ -258,7 +258,9 @@ func (r *ReconcileDependencyBuild) handleStateAnalyzeBuild(ctx context.Context,
 		buildRecipes := []*v1alpha1.BuildRecipe{}
 		_, maven := unmarshalled.Tools["maven"]
 		_, gradle := unmarshalled.Tools["gradle"]
-		for _, image := range []string{"quay.io/sdouglas/hacbs-jdk11-builder:latest", "quay.io/sdouglas/hacbs-jdk8-builder:latest", "quay.io/sdouglas/hacbs-jdk17-builder:latest"} {
+		// FIXME: temporarily changed image for gradle to be replaced after <https://github.com/redhat-appstudio/jvm-build-service/pull/106>
```

Comment:
yeah maybe with @stuartwdouglas 's new configmap based configuring https://github.com/redhat-appstudio/jvm-build-service/pull/105 that it will add a maven vs. gradle switch, so that we can vary what we seed the image ref array with ?

Reply:
yeah we discussed this last night after I posted this comment and determined the configmap would not be the path here